Commercial docs (#886)
miurahr/aqtinstall: docs/commercial.rst

=================================
Qt Commercial Installation Commands
=================================

.. important::

>    * **You can use this for commnercial and also open-source Qt versions**

Reviewer: ```suggestion
   * **You can use this for commercial and also open-source Qt versions**
```

   * **You can only install for the OS running the CLI**
   * **All commands require authentication, even for open-source.** The authentication process:

     * It first looks into the folder of your OS where the file ``qtaccount.ini`` can be:

       * Windows: ``C:\Users\<username>\AppData\Roaming\Qt\qtaccount.ini``
       * Linux: ``/home/<username>/.local/share/Qt/qtaccount.ini``
       * MacOS: ``/Users/<username>/Library/Application Support/Qt/qtaccount.ini``

>      This file is automatically created when you use the ``MaintenanceTool`` binary and log in using the GUI. It is also created the first time you use the CLI or any mean to authenticate. You can simply copy the file that you have on your PC to your CI server, it will work across platforms. Just treat it as a password. Refer to `this page <https://doc.qt.io/qt-6/get-and-install-qt-cli.html#providing-login-information>`_ for more details.

Reviewer: ```suggestion
     This file is automatically created when you use the ``MaintenanceTool`` binary and log in using the GUI. It is also created the first time you use the CLI or any mean to authenticate. You can simply copy the file from your PC to the CI server, it will work across platforms. Just treat it as a password. Refer to `this page <https://doc.qt.io/qt-6/get-and-install-qt-cli.html#providing-login-information>`_ for more details.
```
